tests/unit/list-unique-values.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import Vuex from 'vuex';
+
+import ListUniqueValues from '@/components/ListUniqueValues.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('List Unique Value', () => {
+  const DUMMY_UNIQUE_VALUES = [
+    { value: 'A', nbOcc: 10 },
+    { value: 'B', nbOcc: 4 },
+    { value: 'C', nbOcc: 2 },
+  ];
+
+  it('should instantiate with the correct unique values', () => {
+    const wrapper = shallowMount(ListUniqueValues, {
+      propsData: { values: DUMMY_UNIQUE_VALUES, loaded: true },
+    });
+
+    expect(wrapper.exists()).toBeTruthy();
+    const CheckboxWidgetArray = wrapper.findAll('CheckboxWidget-stub');
+    expect(CheckboxWidgetArray.length).toEqual(3);
+    expect(CheckboxWidgetArray.at(0).vm.$props.label).toEqual('A (10)');
+    expect(CheckboxWidgetArray.at(1).vm.$props.label).toEqual('B (4)');
+    expect(CheckboxWidgetArray.at(2).vm.$props.label).toEqual('C (2)');
+  });
+
+  it('should instantiate with all checkbox checked', () => {
```

Comment:
IMO, this is the responsibility of the component that hosts the state "which unique values are selected".
Regarding my previous comment, I don't think it should be this one